curvefs/build: make build failed on CentOS8

**Describe the bug (描述bug)**
make build failed on CentOS8.


## Issue 1:
ERROR: /home/curve/code/curve/curvefs/test/space/BUILD:123:1: C++ compilation of rule '//curvefs/test/space:space_manager_test' failed (Exit 1)
curvefs/test/space/space_manager_test.cpp: In member function 'virtual void curvefs::space::SpaceManagerTest::SetUp()':
curvefs/test/space/space_manager_test.cpp:54:40: error: 'new' of type 'brpc::Server' with extended alignment 64 [-Werror=aligned-new=]
         server_.reset(new brpc::Server());
                                        ^
curvefs/test/space/space_manager_test.cpp:54:40: note: uses 'void* operator new(std::size_t)', which does not have an alignment parameter
curvefs/test/space/space_manager_test.cpp:54:40: note: use '-faligned-new' to enable C++17 over-aligned new support
cc1plus: all warnings being treated as errors
Target //curvefs/test/space:space_manager_test failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 2.121s, Critical Path: 2.00s


## Issue 2:
./curvefs/src/client/fuse_common.h:30:10: fatal error: fuse_lowlevel.h: No such file or directory
 #include <fuse_lowlevel.h>

[curve@localhost curvefs]$ ls /usr/include/fuse3/fuse_lowlevel.h -l
-rw-r--r--. 1 root root 62440 Nov 14  2017 /usr/include/fuse3/fuse_lowlevel.h

we should change the libary path in curvefs/src/client/BUILD and curvefs/test/client/BUILD.

**To Reproduce (复现方法)**


**Expected behavior (期望行为)**


**Versions (各种版本)**
OS:
Compiler:
curve-mds:
curve-chunkserver:
curve-snapshotcloneserver:
curve-sdk:
nebd:
curve-nbd:

**Additional context/screenshots (更多上下文/截图)**

--- a/curvefs/README.md
+++ b/curvefs/README.md
@@ -22,6 +22,8 @@ Requirement
 Quick Start
 ===
 
+NOTE: If you are using CentOS-8, you can refer to this issue to compile CurveFS: https://github.com/opencurve/curve/issues/807
+
 step 1: clone repository, run build and install:
 
 ```

